pkgs/development/libraries/mesa/stubs.nix

```diff
@@ -1,50 +1,34 @@
 { stdenv
+, lib
 , libglvnd
-, mesa
 , OpenGL
 , testers
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "libGL";
-  inherit (if stdenv.hostPlatform.isDarwin then mesa else libglvnd) version;
+  version = if stdenv.hostPlatform.isDarwin then "4.1" else libglvnd.version;
   outputs = [ "out" "dev" ];
 
-  # On macOS, libglvnd is not supported, so we just use what mesa
-  # build. We need to also include OpenGL.framework, and some
-  # extra tricks to go along with. We add mesa’s libGLX to support
-  # the X extensions to OpenGL.
+  # On macOS, libglvnd is not supported, and mesa no longer builds (but it only
```

Comment:
We'd like to still support mesa for X11 apps, it should be building at the moment.

Reply:
It is, in fact, not building on 24.0.1, _again_, which is how we got to this PR in the first place.

Reply:
Okay. Thanks for heads up, I will need to look.

Reply:
We've discussing it a bit more on Matrix FWIW, though so far to no success.

Reply:
Is there a way to build only the accelerated GLX implementation and not the software renderer? How is libGLX handled on Linux?